src/h5web/providers/h5grove/h5grove-api.ts

```diff
@@ -18,20 +18,15 @@ import type {
   H5GroveAttribute,
 } from './models';
 import { assertDataset } from '../../guards';
-import {
-  convertDtype,
-  flattenValue,
-  handleAxiosError,
-  encodeQueryParams,
-} from '../utils';
+import { convertDtype, flattenValue, handleAxiosError } from '../utils';
 import { buildEntityPath } from '../../utils';
 
 export class H5GroveApi extends ProviderApi {
   protected attrValuesCache = new Map<string, H5GroveAttrValuesResponse>();
 
   /* API compatible with h5grove@0.0.4 */
   public constructor(url: string, filepath: string) {
-    super(filepath, { baseURL: url });
+    super(filepath, { baseURL: url, params: { file: filepath } });
```

Comment:
Slight improvment: the same way that `domain` is set in the HsdsApi constructor, `file` can be set in the constuctor of `H5GroveApi` to avoid setting it for each fetch.

Reply:
nice !!